megamek/src/megamek/server/Server.java

```diff
@@ -887,6 +887,16 @@ private void receivePlayerVersion(Packet packet, int connId) {
             System.out.println("ERROR: Client/Server Version Mismatch -- Client: "+version+" Server: "+MegaMek.VERSION);
             needs = true;
         }
+        else if (clientChecksum == null) {    // print a message indicating client doesn't have jar file
```

Comment:
I haven't understood whether I need to make this change or not, or exactly what the change is.   Do I need the                 buf.append(System.lineSeparator());?   Should the null checks go after the checksum mismatch block?